### ISSUE TITLE
TypeError: ('Keyword argument not understood:', 'output_shape')

### ISSUE THREAD

**Issue by hawkeyedesi** [ellisdg/3DUnetCNN#80]
I tried loading the tumor_segmentation_model.h5. I tried it with Python 2.7.6 and Python 3.4.3 both with Keras V2 with keras-contrib but I keep getting the same error. 

Any suggestions on how to debug this? 

Thanks.

**Comment by ellisdg** [comment]
what version of Keras 2 are you using?

**Comment by hawkeyedesi** [comment]
Keras 2.1.3 and keras-contrib 2.0.8

**Comment by ellisdg** [comment]
I updated the linked model, can you download it and try loading it again?

**Comment by hawkeyedesi** [comment]
Works. Tried both the load_old_model() and set up the network and then a load_weight() with the pre-trained weights. I am able to read them both. 

So what was the problem with the earlier model that it was asking for the output_shape? 

Thanks again.

**Comment by ellisdg** [comment]
Awesome! I'm glad both work. I actually don't know why the wasn't loading previously. I was able to load the model on one machine, but not able to on my personal computer, even though both had identical versions of Keras. I ended up just exporting the weights and then loading them into a newly constructed model. For some reason, that fixed it for now. I figured I would upload the weights as well just in case somebody else runs into a similar issue.
